### PR TITLE
Wireframe

### PR DESCRIPTION
**Pull request by Flunavgt**
On this PR:

set up wireframe
set two components:

- Books
- Categories

### CODE REVIEW

File: src/components/App/App.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import Categories from '../Categories/Categories';
+import NavBar from '../NavBar/NavBar';
+
+const App = () => (
+  <div className="mymain">
+    <BrowserRouter>
+      <NavBar />
+      <Routes>
+        <Route path="/" element={<Books />} />
+        <Route path="/Books" element={<Books />} />
```
Comment (DammyShittu) [inline src/components/App/App.js]: - [ ] There is no need to implement repetitive routes here. The first path "/" should work just fine. Kindly remove the second one "/Books"

File: src/components/NavBar/NavBar.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import './NavBar.css';
+
+const NavBar = () => (
+  <div className="containerNavBar">
+    <div className="NavBar">
+      <div>
+        <h1>Bookstore CMS</h1>
+      </div>
+      <div className="links">
+        <Link to="/Books" className="books">Books</Link>
```
Comment (DammyShittu) [inline src/components/NavBar/NavBar.jsx]: - [ ] Kindly use the root path for your routing here to keep it uniform with the app route.

  ```
  <Link to="/" className="books">Books</Link>
  ```

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @Flunavgt,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lordkaito — APPROVED** [review]
Hi @Flunavgt, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**